modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -2084,6 +2084,62 @@ definition {
 		}
 	}
 
+	@description = "Verify that a label “Parent/Child” is displayed when editing the Relationship"
+	@priority = 3
+	test CanSeeLabelWhenEditingRelationship {
+		task ("Given: An object related to itself, in a Many-to-Many relationship") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 163659",
+				objectName = "CustomObject163659",
+				pluralLabelName = "Custom Objects 163659");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject163659");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject163659",
+				objectName_2 = "CustomObject163659",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "manyToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject163659");
+		}
+
+		task ("When: User accesses Relationships from the Relationship tab") {
+			ObjectAdmin.openObjectAdmin();
```

Comment:
Do we need this step here? It feels redundant. If we're on the Object Admin page, we can just go to the custom object. If we're already viewing the custom object, then we just need to click the Relationships tab.

Reply:
Yes, we need it! Because until now we have done all the initial configurations via API.